src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -172,6 +172,10 @@
     public static final String SIZE_X = "mainWindowSizeX";
     public static final String POS_Y = "mainWindowPosY";
     public static final String POS_X = "mainWindowPosX";
+    public static final String SIZE_Y_CORE = "mainWindowSizeYCore";
```

Comment:
As the user can never configure these values, there is no need to add them as preferences. Please convert them to static fields in `JabRefGUI`.

Reply:
I will try to set the Numbers right in at the method openWindow(), not sure if that works, but if it do I can delete all variables I made up..